src/index.html

```diff
@@ -2,16 +2,164 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="container">
+          <label for="sName">Surname:</label>
+          <input
+            type="text"
+            id="sName"
+            name="sName"
+            autocomplete="off"
+          >
+        </div>
+        <div class="container">
+          <label for="fName">Name:</label>
+          <input
+            type="text"
+            id="fName"
+            name="fName"
+            autocomplete="off"
+          >
+        </div>
+        <div class="container">
+          <label for="age">How old are You?</label>
```

Comment:
when you have attributes inside opening tag write text from new line
```suggestion
          <label for="age">
             How old are You?
          </label>
```